frontend/pages/abandoned/[aid].tsx

```diff
@@ -0,0 +1,370 @@
+import React, { useState, useEffect } from 'react';
+import type { NextPage } from 'next';
+import Link from 'next/link';
+import {
+  StarTwoTone,
+  LikeTwoTone,
+  UserOutlined,
+  PlusCircleOutlined,
+} from '@ant-design/icons';
+import styles from '../burrow/burrow.module.css';
+import { TextLoop } from 'react-text-loop-next';
+import {
+  Alert,
+  Layout,
+  Menu,
+  Breadcrumb,
+  List,
+  Space,
+  message,
+  Form,
+  Button,
+  Input,
+  Card,
+  Tag,
+  Col,
+  Dropdown,
+  Row,
+} from 'antd';
+import { MessageOutlined, LikeOutlined, StarOutlined } from '@ant-design/icons';
+import { useRouter } from 'next/router';
+import 'antd/dist/antd.css';
+import axios, { AxiosError } from 'axios';
+
+axios.defaults.withCredentials = true;
+axios.defaults.headers.post['Content-Type'] = 'application/json';
+
+const { Header, Content, Footer } = Layout;
+const { TextArea } = Input;
+
+const IconText = (props: any) => (
+  <Space>
+    {React.createElement(props.icon)}
+    {props.text}
+  </Space>
+);
+
+function showtag1(tag: string, index: number) {
+  return <Tag key={index}>{tag}</Tag>;
+}
+const showtag = (value: Array<string>) => {
+  return value.map(showtag1);
+};
+
+const Abandoned: NextPage = () => {
+  const initialchange1 = new Array(10).fill(false);
+  const initialchange2 = new Array(10).fill(false);
+  const initialnum1 = new Array(10).fill(0);
+  const initialnum2 = new Array(10).fill(0);
+
+  const [listData, setListData] = useState([]);
+  const [description, setDescription] = useState('Welcome!');
+  const [burrowTitle, setBurrowTitle] = useState(0);
+  const [page, setPage] = useState(1);
+  const [descriptionTemp, setDescriptionTemp] = useState('');
+
+  const [changeLike, setChangeLike] = useState(initialchange1);
+  const [changeCol, setChangeCol] = useState(initialchange2);
+  const [likeNum, setLikeNum] = useState(initialnum1);
+  const [colNum, setColNum] = useState(initialnum2);
+
+  const router = useRouter();
+  const { aid } = router.query;
+  const site = router.pathname.split('/')[1];
+  const [menuMode, setMenuMode] = useState<'inline' | 'horizontal'>(
+    'horizontal'
+  );
+
+  useEffect(() => {
+    try {
+      const fetchListData = async () => {
+        const res = await axios.get('http://127.0.0.1:4523/mock/435762/1');
+        const postlist = await res.data;
+        setListData(postlist.posts);
+        setDescription(postlist.description);
+        setBurrowTitle(postlist.title);
+      };
+      fetchListData();
+    } catch (e) {
+      const err = e as AxiosError;
+      if (err.response?.status === 400) {
+        message.info('请先登录！');
+        router.push('/login');
+      } else if (err.response?.status === 500) {
+        message.info('服务器错误！');
+        router.push('/404');
```

Comment:
refresh page instead of jump to 404 page.
404 page should be only use for status 404，and it will automatically jump.